### PR TITLE
Added new prop in order to set the default position.

### PR DESCRIPTION
**Pull request by kikoruiz**
This PR solves this: https://github.com/SUI-Components/sui-autocompleted/issues/19

Thanks @arturoscmspain. Please @SUI-Components/developers review!

### CODE REVIEW

File: docs/autocompleted-container.jsx [hunk 1]
```diff
@@ -31,10 +31,12 @@ export default class AutocompletedContainer extends React.Component {
...
       <Autocompleted
-          placeholder='Programming Laguages'
-          handleChange={this.handleChange.bind(this)}
-          handleSelect={this.handleSelect.bind(this)}
-          suggests={this.state.suggests}/>
+        placeholder='Programming Laguages'
+        handleChange={this.handleChange.bind(this)}
+        handleSelect={this.handleSelect.bind(this)}
+        suggests={this.state.suggests}
+        defaultPosition={-1}
```
Comment (ruben-martin-lozano) [inline docs/autocompleted-container.jsx]: This approach would solve the issue, but in a more conceptual way, I don't see in wich scenario I would like to set the defaultPosition to something different to 0 or -1 (for example, the third item).

I think would make more sense to solve it via a non-required boolean prop, that says "isFirstItemSelected", by default to true to not to break the current developments in other projects.

But anyway, this is just another way to solve it, so for this is ok 👍

**Comment by kikoruiz** [comment]
Added some refactor, mainly moved const functions into class methods.

**Comment by ruben-martin-lozano** [comment]
:+1: excellent!

**Comment by midudev** [comment]
👍

**Comment by midudev** [comment]
👍 👍

**Pull request merged**